code/game/gamemodes/cult/structures/altar.dm

```diff
@@ -0,0 +1,49 @@
+/obj/structure/cult/talisman
+	name = "daemon altar"
+	desc = "A bloodstained altar dedicated to Nar-Sie."
+	description_antag = "If you are a cultist, you could click on this altar to pray to Nar'Sie, who will in turn heal some of your ailments."
+	icon_state = "talismanaltar"
+	var/last_use = 0
+
+/obj/structure/cult/talisman/examine(mob/user)
+	..(user)
+	if(!iscultist(user) || !isobserver(user))
+		desc = "A bloodstained altar. Looking at it makes you feel slightly terrified."
+	else
+		desc = "A bloodstained altar dedicated to Nar-Sie."
+
+/obj/structure/cult/talisman/attack_hand(mob/user)
+	. = ..()
+	if(iscultist(user) && ishuman(user))
+		if(last_use <= world.time)
+			last_use = world.time + 1200 // Cooldown of two minutes
+			var/mob/living/carbon/human/H = user
+			H.heal_overall_damage(20, 20)
+			to_chat(H, span("cult", "You begin your prayer to Nar'Sie, your wounds slowly closing up.."))
+			for(var/obj/item/organ/external/O in H.organs)
+				if(O.status & ORGAN_ARTERY_CUT)
+					to_chat(H, span("warning", "Severed artery found in [O.name], repairing.."))
```

Comment:
This will flow really weirdly. "Severe artery found in right arm." Make it a bit cooler and more IC, please.

Also, three dots for the ellipsis.

Reply:
"Severed arty", not severe. Putting dots in now.